src/codeenginestudio/huong/Exercise1.java

```diff
@@ -0,0 +1,47 @@
+package codeenginestudio.huong;
+
+import java.util.Arrays;
+
+public class Exercise1 {
+    public static void main(String[] args) {
+
+        String[] strArray = {"is2", "pa6" ,"Thi1s", "T4est", "3a", "th5"};
```

Comment:
They will give us a String as an input! Please write a method to slice a string into an array!